staging/src/kubevirt.io/api/instancetype/v1beta1/types.go

```diff
@@ -519,6 +519,11 @@ type DevicePreferences struct {
 	//
 	// +optional
 	PreferredInterfaceMasquerade *v1.InterfaceMasquerade `json:"preferredInterfaceMasquerade,omitempty"`
+
+	// PreferredPanicDevices optionally defines any panic devices to be used to signal QEMU guest failures.
```

Comment:
`// PreferredPanicDevices optionally defines any panic devices to be used to signal guest failures.`